doc/contributing/collaborator-guide.md

```diff
@@ -168,7 +171,23 @@ dismiss the objection.
 Pull requests with outstanding objections must remain open until all
 objections are satisfied. If reaching consensus is not possible, a
 collaborator can escalate the issue to the TSC by pinging `@nodejs/tsc` and
-adding the `tsc-agenda` label to the issue.
+adding the `tsc-agenda` label to the issue. The change cannot proceed without
+either reaching consensus or a TSC decision to dismiss the objection(s). If the
+TSC does choose to dismiss any objections, a clear explanation of the reasoning
+must be given in the pull request before it lands.
+
+All Collaborator objections are considered equal. There is no greater weight given to
+objections from TSC members than from any other Collaborator.
+
+Mistakes do happen from time-to-time. If a pull request is merged with an
+outstanding objection, corrective action should be taken. Any Collaborator
```

Comment:
Could we make this stronger by stating it is an obligation if pressed?

```suggestion
outstanding objection, corrective action must be taken if requested. Any Collaborator
```

Reply:
Here I think "should be taken" is better. The "must be taken if requested" ends up meaning the same thing ultimately. We could upgrade this to just say, "corrective action must be taken.", which hopefully would satisfy more of the concerns on this. 

Specifically, I think we're all in agreement that corrective action in these cases is a requirement. I think where we disagree is about exactly what form that correction action takes. Some say it *must* be a revert while I'm arguing that a revert is only one of the options.